core/storage/topic/gae_models.py

```diff
@@ -166,6 +166,10 @@ class TopicModel(base_models.VersionedModel):
     # Multiply and Divide'.
     page_title_fragment_for_web = datastore_services.StringProperty(
         required=True, indexed=True)
+    # These skills Id will be used to generate the diagnostic test that
+    # determines whether to recommend this topic to new learners.
```

Comment:
```suggestion
    # These skill IDs will be used to generate the diagnostic test. The diagnostic test is 
    # then used to determine whether the topic is useful to a new learner.
```

Reply:
Not sure if I understand this correctly or not.

Reply:
Updated PTAL.